### ISSUE TITLE
`useSetUID` has no default value

### ISSUE THREAD

**Issue by vandenoever** [NixOS/nixpkgs#19087]
## Issue description

`useSetUID` has no default value

### Steps to reproduce

```bash
$ nix-env -f. -qaP
error: anonymous function at /home/user/nixpkgs/pkgs/os-specific/linux/pam_usb/default.nix:1:1 called without required argument ‘useSetUID’, at /home/user/nixpkgs/lib/customisation.nix:56:12
```

## Technical details

* System: (NixOS: `nixos-version`, Ubuntu/Fedora: `lsb_release -a`, ...)
16.03.1299.a8e0739 (Emu)
* Nix version: (run `nix-env --version`)
nix-env (Nix) 1.11.2
* Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`)
423eebde88f277c4218b68931b7577ca187ce6a6

**Comment by siddharthist** [comment]
I am also experiencing this. I cannot install any packages with `nix-env -i` without getting the exact same error.

**Comment by edolstra** [comment]
Fixed in fe9e5f9f55c461d6722111e5aaa4ccfb2d13ca19.
